examples/complete/main.tf

```diff
@@ -65,12 +65,14 @@ module "trusted_profile" {
   trusted_profile_description = "Example Trusted Profile"
 
   trusted_profile_policies = [{
+    name  = "Policy-1"
     roles = ["Reader", "Viewer"]
     resources = [{
       resource_group_id = module.resource_group.resource_group_id
       service           = "kms"
     }]
     }, {
+    name  = "Policy-2"
```

Comment:
If you try changing this to `${var.prefix}-profile-1` it might pass the upgrade test